pyfstat/make_sfts.py

```diff
@@ -238,9 +238,12 @@ def basic_setup(self):
     def tend(self):
         return self.tstart + self.duration
 
-    def make_data(self):
+    def make_data(self, verbose=False):
         """ A convienience wrapper to generate a cff file then sfts """
-        self.make_cff()
+        if self.h0 == 0:
+            logging.info("Got h0=0, not writing an injection .cff file.")
+        else:
+            self.make_cff(verbose)
```

Comment:
This `verbose` option doesn't apply if `self.h0==0`.

I'd say that's not a problem at all.

Reply:
Well yes, in that case the make_cff is now no longer called at all, verbose or not.